concepticondata/conceptlists/Saxena-2013-157.tsv

```diff
@@ -0,0 +1,158 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Saxena-2013-157-1	1	maternal grandfather	2178	MATERNAL GRANDFATHER
+Saxena-2013-157-2	2	maternal grandmother	2177	MATERNAL GRANDMOTHER
+Saxena-2013-157-3	3	paternal grandfather	2560	PATERNAL GRANDFATHER
+Saxena-2013-157-4	4	paternal grandmother	3015	PATERNAL GRANDMOTHER
+Saxena-2013-157-5	5	wife	1199	WIFE
+Saxena-2013-157-6	6	mother	1216	MOTHER
+Saxena-2013-157-7	7	daughter	1357	DAUGHTER
+Saxena-2013-157-8	8	older sister	1758	OLDER SISTER
+Saxena-2013-157-9	9	younger sister	1761	YOUNGER SISTER
+Saxena-2013-157-10	10	husband	1200	HUSBAND
+Saxena-2013-157-11	11	father	1217	FATHER
+Saxena-2013-157-12	12	son	1620	SON
+Saxena-2013-157-13	13	older brother	1759	OLDER BROTHER
+Saxena-2013-157-14	14	younger brother	1760	YOUNGER BROTHER
+Saxena-2013-157-15	15	maternal uncle	3220	MATERNAL UNCLE
+Saxena-2013-157-16	16	maternal aunt	2176	MATERNAL AUNT
+Saxena-2013-157-17	17	paternal uncle		
+Saxena-2013-157-18	18	paternal aunt	2175	PATERNAL AUNT
+Saxena-2013-157-19	19	head	1256	HEAD
+Saxena-2013-157-20	20	face	1560	FACE
+Saxena-2013-157-21	21	hair (head)	2648	HAIR (HEAD)
+Saxena-2013-157-22	22	tail	1220	TAIL
+Saxena-2013-157-23	23	ear	1247	EAR
+Saxena-2013-157-24	24	eye	1248	EYE
+Saxena-2013-157-25	25	nose	1221	NOSE
+Saxena-2013-157-26	26	mouth	674	MOUTH
+Saxena-2013-157-27	27	tooth	1380	TOOTH
+Saxena-2013-157-28	28	hand	1277	HAND
+Saxena-2013-157-29	29	foot	1301	FOOT
+Saxena-2013-157-30	30	animal	619	ANIMAL
+Saxena-2013-157-31	31	autumn	1276	AUTUMN
+Saxena-2013-157-32	32	bird	937	BIRD
+Saxena-2013-157-33	33	blood	946	BLOOD
+Saxena-2013-157-34	34	bone	1394	BONE
+Saxena-2013-157-35	35	butter	1245	BUTTER
+Saxena-2013-157-36	36	cat (f,m)	1208	CAT
+Saxena-2013-157-37	37	child	2099	CHILD
+Saxena-2013-157-38	38	cloud	1489	CLOUD
+Saxena-2013-157-39	39	copper	1825	COPPER
+Saxena-2013-157-40	40	day	1225	DAY (NOT NIGHT)
+Saxena-2013-157-41	41	dog	2009	DOG
+Saxena-2013-157-42	42	egg	744	EGG
+Saxena-2013-157-43	43	fire	221	FIRE
+Saxena-2013-157-44	44	food	256	FOOD
+Saxena-2013-157-45	45	forest	420	FOREST
+Saxena-2013-157-46	46	fruit	1507	FRUIT
+Saxena-2013-157-47	47	glacier		
+Saxena-2013-157-48	48	goat	1502	GOAT
+Saxena-2013-157-49	49	gold	1369	GOLD
+Saxena-2013-157-50	50	grass	606	GRASS
+Saxena-2013-157-51	51	hen	1514	HEN
+Saxena-2013-157-52	52	iron	621	IRON
+Saxena-2013-157-53	53	lamb	820	LAMB
+Saxena-2013-157-54	54	leaf	628	LEAF
+Saxena-2013-157-55	55	man (human)	683	PERSON
+Saxena-2013-157-56	56	man (male)	1554	MAN
+Saxena-2013-157-57	57	meat	2615	FLESH OR MEAT
```

Comment:
I think this could be mapped to 634 MEAT. The glosses mapped to FLESH OR MEAT mostly look like this: _meat/flesh_

Reply:
This is most likely due to this being based on Swadesh-1955-215, which links to FLESH OR MEAT as well.

Reply:
Thank you for your review, @AnnikaTjuka! I checked the concepts that were mapped to a broader concept set again, and as Christoph already said, they are mapped that way in Swadesh, so I'll just leave them like that. Although I agree that especially rain could probably be the narrower concept here given that there are no verbs in this list.